ctapipe/reco/impact_distance.py

```diff
@@ -57,9 +58,10 @@ def shower_impact_distance(
     )
 
     # sky direction of the shower (defines the shower axis)
-    sky_direction = SkyCoord(
-        alt=shower_geom.alt, az=shower_geom.az, frame="altaz"
-    ).cartesian.xyz.value
+    # NOTE: remember that in sky direction the azimuth needs to be negative
+    sky_direction = spherical_to_cartesian(
```

Comment:
I think at this point we should define a `altaz_to_righthanded_cartesian` in `ctapipe.coordinates`.